src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -168,36 +169,23 @@ describe('manage nsfs cli bucket flow', () => {
         });
 
         it('should fail - cli create bucket - owner is an IAM account', async () => {
-            const { name } = account_defaults;
-            const accounts_details = await read_config_file(config_root, CONFIG_SUBDIRS.ACCOUNTS, name);
-            const account_id = accounts_details._id;
-
             const { new_buckets_path: account_path } = account_defaults;
             const account_name = 'account-to-be-owned';
-            const account_options1 = { config_root, name: account_name, uid: 5555, gid: 5555,
-                new_buckets_path: account_path};
+            const account_options1 = { config_root, name: account_defaults.name, uid: 5555, gid: 5555,
+                new_buckets_path: account_path, iam_name: account_name};
             await fs_utils.create_fresh_path(account_path);
             await fs_utils.file_must_exist(account_path);
             await set_path_permissions_and_owner(account_path, account_options1, 0o700);
             await exec_manage_cli(TYPES.ACCOUNT, ACTIONS.ADD, account_options1);
 
-            // update the account to have the property owner
-            // (we use this way because now we don't have the way to create IAM users through the noobaa cli)
-            const account_config_path = path.join(config_root, CONFIG_SUBDIRS.ACCOUNTS, account_name + '.json');
-            const { data } = await nb_native().fs.readFile(DEFAULT_FS_CONFIG, account_config_path);
-            const config_data = JSON.parse(data.toString());
-            config_data.owner = account_id; // just so we can identify this account as IAM user;
-            await update_config_file(DEFAULT_FS_CONFIG, CONFIG_SUBDIRS.ACCOUNTS,
-                account_config_path, JSON.stringify(config_data));
-
             // create the bucket
             const action = ACTIONS.ADD;
             const bucket_options = { config_root, ...bucket_defaults, owner: account_name};
             await fs_utils.create_fresh_path(bucket_options.path);
             await fs_utils.file_must_exist(bucket_options.path);
             await set_path_permissions_and_owner(bucket_options.path, { uid: account_options1.uid, gid: account_options1.gid }, 0o700);
             const res = await exec_manage_cli(TYPES.BUCKET, action, bucket_options);
-            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.BucketSetForbiddenBucketOwnerIsIAMAccount.code);
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.BucketSetForbiddenBucketOwnerNotExists.code);
```

Comment:
If you think we don't need this test anymore, you can remove it.
If you leave it, please add comments, the result is `BucketSetForbiddenBucketOwnerNotExists` because the owner only checks for root accounts, and we passed an IAM account.
If you think that we can remove this error after the config change, I think you can remove.

Reply:
I think it's a good test - it verifies correct behavior in case user did something that is plausible.

Reply:
Now we will not have the error 'BucketSetForbiddenBucketOwnerIsIAMAccount' thrown in any case, right?
I think you can remove it.
I don't mind leaving the test, please just add comments since it was changed a bit.

Reply:
Added comment to explain the error code.